pennylane/ops/qubit.py

```diff
@@ -73,6 +73,11 @@ def diagonalizing_gates(self):
         """
         return [RY(-np.pi / 4, wires=self.wires)]
 
+    @staticmethod
+    def decomposition(wires):
+        decomp_ops = [RX(np.pi / 2, wires=wires), RZ(np.pi / 2, wires=wires)]
```

Comment:
Unless I'm looking at it incorrectly, this doesn't seem to work for me 🤔 
```python
>>> qml.RZ(np.pi/2, wires=0).matrix @ qml.RX(np.pi/2, wires=0).matrix
array([[ 0.5-0.5j, -0.5-0.5j],
       [ 0.5-0.5j,  0.5+0.5j]])
```

However, a 180 degree Y rotation followed by a 90 degree X rotation works:

```python
>>> qml.PauliX(0).matrix @ qml.RY(np.pi/2, wires=0).matrix
array([[ 0.70710678,  0.70710678],
       [ 0.70710678, -0.70710678]])
```

Reply:
Yes, you're right, I meant to do Rz(pi/2) Rx(pi/2) Rz(pi/2), but the one you provide in more simple 🙂